.github/workflows/publish_release.yml

```diff
@@ -7,17 +7,19 @@ on:
   push:
     branches:
       - release-*
+  release:
```

Comment:
Please update the first line of this file to `name: "publish release - npm, driver, trace viewer"`.